docs/quickstart.md

```diff
@@ -0,0 +1,428 @@
+
+### bundle - combine several definition files into a single one
+
+With this command, you can merge standalone files back into a single definition file. `bundle` does the opposite of the [`split`](#split) command.
+
+Let's bundle the definition file that you split earlier.
+
+In your terminal, type the following:
+
+```bash
+$ openapi bundle openapi/openapi.yaml --output bundled.yaml
+```
+
+You should get the following response:
+
+```bash
+$ openapi bundle openapi/openapi.yaml --output bundled.yaml
+bundling openapi/openapi.yaml...
+📦 Created a bundle for openapi/openapi.yaml at bundled.yaml 46ms.
+```
+
+You should now see a new `bundled.yaml` file as a result of the `bundle` command completion. Open this file in your text editor or IDE and compare it with the linted definition file. You can see that both files are the same which means that the `bundle` command executed flawlessly.
+
+### preview-docs - preview your definition files quickly
+
+With this command, you can generate a preview your OpenAPI definition file.
+
+In your terminal, type the following:
+
+```bash
+$ openapi preview-docs petstore.yaml 
+```
+
+You should get the following response:
+
+```bash
+$ openapi preview-docs petstore.yaml 
+Using Redoc community edition.
+Login with openapi-cli login or use an enterprise license key to preview with the premium docs.
+
+
+  🔎  Preview server running at http://127.0.0.1:8080
+
+Bundling...
+
+
+  👀  Watching petstore.yaml and all related resources for changes
+
+Created a bundle for petstore.yaml successfully
+```
+
+Navigate to `http://127.0.0.1:8080` and check that your definition file has been served successfully:
+
+![preview-docs](../images/preview-docs.png)
+
+This server support live changes - try to modify your definition file, these modifications will be previewed immediately.
+
+For example, change the version of your Swagger Petstore from `1.0.0` to `1.1.0`:
+
+```yaml petstore.yaml
+openapi: "3.0.0"
+info:
+  version: 1.1.0
+  ...
+```
+
+You should get the following response:
+
+```bash
+watch changed petstore.yaml
+watch changed /path-to-your-project/petstore.yaml
+
+Bundling...
+
+Created a bundle for petstore.yaml successfully
+GET /: 4.337ms
+GET /: 3.825ms
+GET /hot.js: 27.698ms
+GET /simplewebsocket.min.js: 42.654ms
+GET /: 6.77ms
+GET /hot.js: 1.639ms
+GET /simplewebsocket.min.js: 14.528ms
+GET /simplewebsocket.min.js: 1.47ms
+GET /hot.js: 1.826ms
+GET /openapi.json: 0.499ms
+GET /openapi.json: 2.293ms
+GET /openapi.json: 0.867ms
+```
+
+Navigate to `http://127.0.0.1:8080` and check the updated definition file:
+
+![updated-preview-docs](../images/updated-preview-docs.png)
+
+## Next steps
+
+* Check the full list of [OpenAPI CLI commands](./commands/index.md) available to you
+* Learn how to fine-tune your OpenAPI CLI tool using [configuraton file](./configuration/index.mdx)
+* Learn more about [custom plugins and rules](./custom-rules.md)
```

Comment:
```suggestion
- Check the full list of [OpenAPI CLI commands](./commands/index.md) available to you
- Learn how to fine-tune your OpenAPI CLI tool using [configuraton file](./configuration/index.mdx)
- Learn more about [custom plugins and rules](./custom-rules.md)
```
We didn't share this info but we're trying to consistently use the hyphen for bullets in Markdown.